src/data-types.js

```diff
@@ -87,6 +87,18 @@ export function getTypes(options) {
 		}
 	}
 
+	var int64 = {
+		unpack: function(buffer){
+			var value = buffer.readUInt64LE(buffer.pointer);
```

Comment:
@mrfishie Well, the Gobble build system does check a few things when doing `npm install` (transpiles ES6 to ES5), but this is one of those cases where unit tests (one per packet type) should come **very** handy.